tests/unit/auth/v2/queryAuthCheck.js

```diff
@@ -58,20 +60,50 @@ describe('v2: queryAuthCheck', () => {
 
     clock.tick(604800000);  // take time 604800000ms (7 days) ahead
 
-    it('URL should expire after 7 days with default preSignUrlExpiry', () => {
+    it('URL should expire after 7 days with default expiry', () => {
         const currentTime = Date.now();
         const request = { method: 'GET', query: { Expires: currentTime } };
         const data = { Expires: currentTime };
         const res = queryAuthCheck(request, log, data);
         assert.notStrictEqual(res.err, undefined);
         assert.strictEqual(res.err.AccessDenied, true);
     });
-    it('should raise MissingSecurityHeader with invalid expires param', () => {
-        const request = { method: 'GET', query: { Expires: 'a string' } };
-        const data = { Expires: 'a string' };
+
+    clock.restore();
```

Comment:
this might need / should be in an afterEach or afterAll, whatever the test framework has I cant remember.

Reply:
also done